coalib/output/ConsoleInteraction.py

```diff
@@ -152,14 +153,33 @@ def acquire_actions_and_apply(console_printer,
             metadata_list.append(metadata)
 
         # User can always choose no action which is guaranteed to succeed
-        if not ask_for_action_and_apply(console_printer,
-                                        section,
-                                        metadata_list,
-                                        action_dict,
-                                        failed_actions,
-                                        result,
-                                        file_diff_dict,
-                                        file_dict):
+        if section.name == 'cli':
+            if not ask_for_action_and_apply(console_printer,
+                                            section,
+                                            metadata_list,
+                                            action_dict,
+                                            failed_actions,
+                                            result,
+                                            file_diff_dict,
+                                            file_dict):
+                break
+            question = format_lines(
+                'Reprompt available actions (True/ False)? ', symbol='!')
+            input_q = input(question)
+            if input_q == 'True' or input_q == 'False':
```

Comment:
Is there a better way to do this? I know that something like SpaceConsistencyBear can accept Yes, No, Yeah, etc